docs/self-hosted/config-options.md

```diff
@@ -952,16 +952,16 @@ Based on the `EMAIL_TRANSPORT` used, you must also provide the following configu
 
 ### SMTP (`smtp`)
 
-| Variable                | Description      | Default Value |
-| ----------------------- | ---------------- | ------------- |
-| `EMAIL_SMTP_NAME`       | SMTP Hostname    | --            |
-| `EMAIL_SMTP_HOST`       | SMTP Host        | --            |
-| `EMAIL_SMTP_PORT`       | SMTP Port        | --            |
-| `EMAIL_SMTP_USER`       | SMTP User        | --            |
-| `EMAIL_SMTP_PASSWORD`   | SMTP Password    | --            |
-| `EMAIL_SMTP_POOL`       | Use SMTP pooling | --            |
-| `EMAIL_SMTP_SECURE`     | Enable TLS       | --            |
-| `EMAIL_SMTP_IGNORE_TLS` | Ignore TLS       | --            |
+| Variable                | Description          | Default Value |
+| ----------------------- | -------------------- | ------------- |
+| `EMAIL_SMTP_NAME`       | SMTP client hostname | --            |
```

Comment:
Should this option be on top? It is an optional value that should not be filled in most cases (your server hostname should be set correctly in a proper setup)

Reply:
Move to bottom 👍